Objective-C/Internal/Replicator/CBLTrustCheck.mm

```diff
@@ -146,7 +146,7 @@ - (NSURLCredential*) checkTrust: (NSError**)outError {
     // Evaluate trust:
     SecTrustResultType result;
     OSStatus err;
-#if TARGET_OS_MACCATALYST
+#if __MAC_OS_X_VERSION_MIN_REQUIRED >= 101400 || __IPHONE_OS_VERSION_MIN_REQUIRED >= 120000
```

Comment:
I'm trying to understand the problem. Doesn't this have the affect when we build the framework not when the user uses the framework?